.circleci/config.yml

```diff
@@ -222,3 +262,14 @@ workflows:
               only: /.*/
           context:
             - aws-lambda
+      - snyk_test:
+          context:
+            - snyk
+      # Monitor sends to dashboard
+      - snyk_monitor:
+          context:
+            - snyk
+          filters:
+            branches:
+              only: /^develop/
```

Comment:
Does this mean we won't get a heads-up if a PR contains a dependency with a severe vulnerability? 
Or are we leaving in place the current (flawed) dashboard and hope that PR status check catches the worst offenders?

Reply:
> Does this mean we won't get a heads-up if a PR contains a dependency with a severe vulnerability?
Correct. The problem is that not all vulnerabilities are fixable, so I didn't want to block the build.

The other option is to force engineers to update the .snyk file to explicitly allow vulnerabilities.

> Or are we leaving in place the current (flawed) dashboard and hope that PR status check catches the worst offenders?
This should fix the flawed dashboard -- we will push to it via the CLI.

Reply:
> This should fix the flawed dashboard -- we will push to it via the CLI. 

We push to it on develop. I mean on PRs, will we leave the dashboard and github status checks in place so we get at least an approximation

Reply:
> We push to it on develop. I mean on PRs, will we leave the dashboard and github status checks in place so we get at least an approximation

As far as I could tell, the dashboard doesn't show the branch, so I felt it would be confusing for the dashboard to show whatever branch happened to be pushed last.

I'll revisit the whole thing.